app/recurring_expenses/views.py

```diff
@@ -39,53 +40,102 @@ def create_or_edit_recurring_expense(request, expense_id=None):
         edit_recurring_expense(request, expense_id=expense_id)
 
 
+_IMPORT_CONTENT_KEYS = {
+    "json": "application/json",
+    "csv": "text/csv",
+}
+
+_IMPORT_CONTENT_TYPES = {value: key for key, value in _IMPORT_CONTENT_KEYS.items()}
```

Comment:
Constants are fine but in python we don't really prefix them with an ``_``

I would prefer ``_IMPORT_CONTENT_KEYS`` becomes ``IMPORT_CONTENT_KEYS``

Same for any other constants

Reply:
Sure, will rename those.